inst/htmljs/animint.js

```diff
@@ -1692,11 +1691,11 @@ var animint = function (to_select, json_file) {
           selected_ids[i] = v_name.concat("___", s_info.selected[i]);
 	}
       }
-      // update selected widgets, if necessary
-      if(s_info.type == "multiple" | 
-	 selectized_array[v_name].getValue() != selected_ids) {
-	selectized_array[v_name].setValue(selected_ids);
-      }
+      // from
+      // https://github.com/brianreavis/selectize.js/blob/master/docs/api.md:
+      // setValue(value, silent) If "silent" is truthy, no change
+      // event will be fired on the original input.
+      selectized_array[v_name].setValue(selected_ids, true);
```

Comment:
Seems to me like the `if(multiple selection or selectized.getValue is not selected_ids)` is not necessary here. @kferris10 any reason you can think of for keeping this?